src/main/resources/application.yml

```diff
@@ -1,4 +1,10 @@
 spring:
+  jpa:
+    generate-ddl: true
+    hibernate:
+
+      ddl-auto: create
+    show-sql: true
```

Comment:
I assume this is to set spring up to automagically manage our database?
I don't think that is a good idea for prod, see my comment in #31